src/ebonite/core/objects/core.py

```diff
@@ -303,13 +304,15 @@ class Model(EboniteObject):
     :param creation_date: date when this model was created
     """
 
-    def __init__(self, name: str, wrapper: ModelWrapper,
+    def __init__(self, name: str, wrapper: Optional[ModelWrapper] = None,
```

Comment:
In my mind it was wrapper_json that is the field in the constructor, and wrapper only a property. I don't know which is better, but think about it.

Reply:
Proposed solution requires us to serialize model during creation. I think this is overkill and requires us to store JSON stub instead of real object for no reasons. I prefer using JSON only for cases when it is absolutely required.

Reply:
ok